src/gui/systray.cpp

```diff
@@ -285,6 +285,103 @@ void Systray::destroyEditFileLocallyLoadingDialog()
     _editFileLocallyLoadingDialog = nullptr;
 }
 
+bool Systray::raiseDialogs()
+{
+    return raiseFileDetailDialogs();
+}
+
+bool Systray::raiseFileDetailDialogs(const QString &localPath)
+{
+    if(_fileDetailDialogs.empty()) {
+        return false;
+    }
+
+    auto it = _fileDetailDialogs.begin();
+    while (it != _fileDetailDialogs.end()) {
+        const auto dialog = *it;
+        auto liveDialog = dialog != nullptr;
```

Comment:
Somehow this naming `liveDIalog` makes me think there is a live broadcast happening. Maybe some other name could better emphasize its meaning?

Reply:
I changed `liveDialog` to be now `nullDialog`